ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestCase.java

```diff
@@ -410,6 +410,18 @@ private static void validateTopicData(
     }
   }
 
+  private static String getActualsForErrorMessage(final List<FakeKafkaRecord> actual) {
+    final StringBuilder stringBuilder = new StringBuilder("Actual records: \n");
+    for (final FakeKafkaRecord fakeKafkaRecord: actual) {
+      final ProducerRecord<?,?> producerRecord = fakeKafkaRecord.getProducerRecord();
+      stringBuilder.append(" timestamp: ").append(producerRecord.timestamp())
```

Comment:
nit: perhaps we can standardize on a format for printing records? This format here is different from the one used in `validateCreatedMessage()` below to generate error messages for when expected output records don't match. Besides creating consistency from a UX perspective, having the two share a single code-path for printing would also reduce code :)

Reply:
Good point. I created a new method for printing producer records and use it in both cases.